Frontend/src/pages/MyRecipes.tsx

```diff
@@ -1,152 +1,156 @@
 import React from "react";
 import "../style/MyRecipes.css";
-import Box from '@mui/material/Box';
-import Grid from '@mui/material/Grid';
-import Stack from '@mui/material/Stack';
-import Paper from '@mui/material/Paper';
-import { styled } from '@mui/material/styles';
 import Rating from "../components/Rating";
 import InfoTable from "../components/InfoTable";
-import BtnAdvntrz from "../assets/Logos_Icons/Button_Adventurize_it.png"
-import StickyNote from "../assets/fillElements/StickyNote_2.png";
 import Ramen from "../assets/recipes/recipe_Ramen_01_23.jpg";
-import Button from '@mui/material/Button';
+import Stack from "@mui/material/Stack";
+import Item from "@mui/material/Grid";
+import RD from "../helpers/RecipesDB_Simulator";
 
 
-const DemoPaper = styled(Paper)(({ theme }) => ({
-    width: 1500,
-    height:1600,
-    position: 'relative',
-    bottom: '83vh',
-    left: '58vw',
-    padding: theme.spacing(2),
-    ...theme.typography.body2,
-    backgroundColor: '#c7fc70',
-    textAlign: 'center',
-    boxShadow: '4px 4px #000000',
-}));
+// Just a try to get the data from the DB_Fetcher
+ let recipe = RD.map((recipe) => {
+    console.log(recipe);
+    console.log(recipe.name);
+    console.log(recipe.preparationTime);
+    console.log(recipe.cookingTime);
+    console.log(recipe.difficulty);
+    console.log(recipe.ingredients);
+    console.log(recipe.instructions);
+    console.log(recipe.rating);
+    return recipe.toString();
+} );
+
+console.log(recipe);
```

Comment:
pls remove console.log()

Reply:
This command is only there to avoid an unused-variable-error of the recipe array. Can be removed after integration of the component.